components/mailbox/src/Mailbox.svelte

```diff
@@ -101,30 +101,8 @@
       folders = await FolderStore.getFolders(accountOrganizationUnitQuery);
     }
 
-    if (_this.all_threads) {
-      threads = _this.all_threads as Thread[];
-    } else if (_this.keyword_to_search) {
-      threads = await MailboxStore.getThreadsWithSearchKeyword({
-        ...accountOrganizationUnitQuery,
-        keyword_to_search: _this.keyword_to_search,
-      });
-    } else {
-      threads = (await MailboxStore.getThreads(query)) || [];
-    }
+    await updatedDisplayedThreads();
```

Comment:
`updatedDisplayedThreads` did you mean `updateDisplayedThreads` ?

Reply:
great catch, not sure how this isn't causing any errors

Reply:
oh I see, it's like this everywhere - needs to be renamed 👍